db/schema.rb

```diff
@@ -2,8 +2,8 @@
 # of editing this file, please use the migrations feature of Active Record to
 # incrementally modify your database, and then regenerate this schema definition.
 #
-# This file is the source Rails uses to define your schema when running `rails
-# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
```

Comment:
Was this change manually generated?

Reply:
It was auto-generated when I run rails db:setup/seed.

Reply:
That's weird, It's not generating any change for me 🤔

Anyway, I'm fine with keeping it 👍